data/json/overmap/overmap_terrain/overmap_terrain.json

```diff
@@ -205,7 +212,8 @@
     "sym": "F",
     "color": "green",
     "see_cost": 5,
-    "extras": "field"
+    "extras": "field",
+    "flags": [ "RISK_LOW", "SOURCE_OF_SAFETY" ]
```

Comment:
For this and a couple of others like the bandit cabins, it's entirely probable that we're not going to play out the NPC establishing a base here and the first time that the monsters that are here will be active will be when the player brings them into the reality bubble.  It's going to be weird as heck if TonZa has been dropping off his loot in a strange cabin full of shadows without dying, but those same shadows swarm the player when he goes to retrieve TonZa's loot.

For that reason, unless you have a definite plan to make the NPC re-evaluate these places when they get to the, I'd rather they were assigned RISK_EXTREME and not SOURCE_OF_SAFETY so NPCs don't go to them.

Reply:
Would it not be possible that in the future, the NPC does infact play out the establishing a base, and NPCs going to a cabin and dying because another NPC or bandits or monsters are there would be part of that simulation? 

I'd much prefer that, but in the meantime, I can add a form of telepathy for them to avoid places that are dangerous to avoid that incongruity.

Reply:
That's fine to do and I don't even think it's particularly hard to do - there's some half-assed code in mission_companion.cpp that looks like it supports all that.  I'm okay with NPCs dying when trying to set up bases at strange cabins or bandit forts if we're willing to get that code working.  Which is probably a worthwhile endeavor because it's in the same section of the file as the tinymap NPC looting, and we definitely want that working.

Reply:
I'll do that then when I attach up the NPC overmap movement AI to this.